doc/articles/create-an-app-vscode.md

```diff
@@ -121,6 +121,15 @@ You're all set!
 
 You can also find [additional VS Code topics](xref:Uno.vscode.additional), such as using snippets, updating existing apps to use VS Code
 
-## Explore
+## Next Steps
 
-Next, explore [Uno Platform features and architecture](xref:Uno.GetStarted.Explore).
\ No newline at end of file
+Now that you're Created and Debug the App.
+
+Learn more about:
```

Comment:
Maybe something like: "Now that you've created and debugged the App, learn more about:"

Reply:
Made in PR https://github.com/unoplatform/uno/pull/14774